.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: cpp
 sudo: true
-dist: trusty
+dist: bionic
```

Comment:
Is there also a dist tag `latest` as used by ubuntu docker pointing to the latest stable?

Reply:
There does not seem to be, reading their documentation there are 4 options for Ubuntu Linux: `dist: bionic` or `dist: xenial` or `dist: trusty` or `dist: precise`

Reply:
Thanks for checking. Odd they do not have a floating version, but no problem, the DF build will use that and the travis setting will have to be adjusted when needed.